Adyen/adyen-salesforce-commerce-cloud
# Pw 7284/multiple giftcard brands

## Summary

Allow shoppers to use different giftcard brands instead of only one.
New giftcard UI
Modified payment methods UI
Removed modal

## Test scenarios
1. Adding different giftcard brands
2. Removing giftcard
3. Persist giftcard data on refresh and canceled payment after adding
4. Add giftcard -> cancel -> add another giftcard brand
5. Add giftcard -> remove -> add another giftcard brand
6. Add giftcard -> go to home -> add another item to cart -> giftcard should be removed and message should be shown
7. Do payment and try adding giftcard to another payment
